go/base/tasks.py

```diff
@@ -26,6 +26,9 @@ def get_and_reset_recent_conversations(vumi_api):
     redis = vumi_api.redis.sub_manager(
         ConversationMetricsMiddleware.SUBMANAGER_PREFIX)
 
+    # Reset the local cache for recent conversations
+    ConversationMetricsMiddleware.LOCAL_RECENT_CONVS = set()
```

Comment:
This won't actually do anything useful because the middleware is running in an entire different process, probably on another machine. :)